server/Arcadia.Assistant.Web/Authorization/Handlers/UserIsHealthHandler.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Arcadia.Assistant.Web.Authorization.Handlers
+{
+    using System.Threading.Tasks;
+    using Microsoft.AspNetCore.Authorization;
+    using Requirements;
+
+    public class UserIsHealthHandler : AuthorizationHandler<UserIsHealthRequirement>
```

Comment:
no need for handler, use policy instead to select a scheme

Reply:
This is WIP, occasionally got into the PR.